src/python/tests/core/api/test_scheduler.py

```diff
@@ -36,15 +36,15 @@
     await conn.execute('''insert into foglamp.scheduled_processes(name, script)
         values('echo_test', '["echo", "Hello"]')''')
     await conn.close()
-    await asyncio.sleep(4)
+    await asyncio.sleep(14)
 
 async def delete_master_data():
     conn = await asyncpg.connect(database=__DB_NAME)
     await conn.execute('''DELETE from foglamp.tasks WHERE process_name IN ('testsleep30', 'echo_test')''')
     await conn.execute(''' DELETE from foglamp.schedules WHERE process_name IN ('testsleep30', 'echo_test')''')
     await conn.execute(''' DELETE from foglamp.scheduled_processes WHERE name IN ('testsleep30', 'echo_test')''')
     await conn.close()
-    await asyncio.sleep(4)
+    await asyncio.sleep(14)
```

Comment:
is it machine specific or deterministic calculation?

Reply:
Value provided to remove any lag whatsoever due to async processing.